agml/synthetic/synthetic_data_generation/generate.cpp

```diff
@@ -157,7 +170,7 @@ int main(int argc, char** argv) {
         SyntheticAnnotation annotation(&context);
 
         // Choose either the LiDAR or RGB image simulation.
-        if (!config.simulation_type.empty() && config.simulation_type[1] == "lidar") {
+        if (config.simulation_type.size() > 1 && config.simulation_type[1] == "lidar") {
```

Comment:
It is necessary to ensure that the size is greater than 1 here rather than simply checking whether or not the vector is empty. Without this change, the program may encounter a segfault here depending on the compiler.

Reply:
Looks good! thanks.